includes/class-wc-payments-account.php

```diff
@@ -536,9 +538,14 @@ private function redirect_to_login() {
 	private function get_onboarding_return_url( $wcpay_connect_from ) {
 		// If connection originated on the WCADMIN payment task page, return there.
 		// else goto the overview page, since now it is GA (earlier it was redirected to plugin settings page).
-		return 'WCADMIN_PAYMENT_TASK' === $wcpay_connect_from
-			? $this->get_payments_task_page_url()
-			: $this->get_overview_page_url();
+		switch ( $wcpay_connect_from ) {
+			case 'WCADMIN_PAYMENT_TASK':
+				return $this->get_payments_task_page_url();
+			case 'WC_SUBSCRIPTIONS':
```

Comment:
`WC_SUBSCRIPTIONS` is possibly not the best name as it could be confused with the paid extension but I couldn't think of a better name 😓

Reply:
Maybe `WC_SUBSCRIPTIONS_EDIT` or `WC_EDIT_SUBSCRIPTIONS`.

Reply:
I happened to be looking into this just now and the string I used was `WC_SUBSCRIPTIONS_TABLE`. I'm not sure if that's any better but throwing it in the mix.

Reply:
Ah cool! My bad, didn't realise you were looking into this, feel free to decline this PR.

Reply:
Nah I hadn't got to the point of testing it yet so happy to review this one given it's pushed up already.